app/models/recent_activity.rb

```diff
@@ -69,6 +69,12 @@ def self.user_activity(user, reject_keys=[])
   # All activities that are public and should be visible for a user
   # * +user+ - the user which activities will be returned
   def self.user_public_activity user
```

Comment:
Ok, so my suggestion is to leave it as it is for now and I will add a ticket to start using this "recipient" (or something similar) and also improve `RecentActivity.user_public_activity`.